Samples/UnifiedAPI/ArmoniK.Samples.Client/AdditionTest.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using System.Linq;
+using System.Threading;
+
+using ArmoniK.Api.gRPC.V1;
+using ArmoniK.DevelopmentKit.Client.Common;
+using ArmoniK.DevelopmentKit.Client.Common.Exceptions;
+using ArmoniK.DevelopmentKit.Client.Common.Submitter;
+using ArmoniK.DevelopmentKit.Client.Unified.Factory;
+using ArmoniK.DevelopmentKit.Common;
+
+using Google.Protobuf.WellKnownTypes;
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace ArmoniK.Samples.Client
+{
+  internal class AdditionTest : IDisposable
```

Comment:
You do not test addition but the health check so you should rename your files accordingly